ftdc/custom_format.go

```diff
@@ -136,6 +138,11 @@ func isNumeric(kind reflect.Kind) bool {
 }
 
 func flattenStruct(item reflect.Value) ([]float32, error) {
+	_, b, c := flattenStructNew(item)
```

Comment:
The new `flattenStructNew` does everything `flattenStruct` does and more (gets the fields for a schema). To help test correctness of the new code, I've started this PR where everything funnels into the new code.